projects/admin/src/app/menu/menu.component.ts

```diff
@@ -131,6 +131,14 @@ export class MenuComponent implements OnInit {
             routerLink: '/records/persons',
             iconCssClass: 'fa fa-user'
           }]
+        }, {
+          name: this.translateService.instant('Acquisitions'),
```

Comment:
Don't forget to extract the strings for translation: `npm run admin_extract_messages` and `npm run admin_update_catalog`. No need to run tx pull and push command any more.

Reply:
done